foundry/applications/formApplication.d.ts

```diff
@@ -251,31 +251,31 @@ declare namespace FormApplication {
     /**
      * @defaultValue `['form']`
      */
-    classes: string[];
+    classes?: string[];
```

Comment:
Similar to `Application.Options`, these should not be optional. Instead, constructors should take `Partial`s of `FormApplication.Options`.